yt/frontends/ramses/io.py

```diff
@@ -23,6 +23,8 @@
 import yt.utilities.fortran_utils as fpu
 from yt.utilities.lib.cosmology_time import \
     get_ramses_ages
+from yt.utilities.exceptions import YTFieldTypeNotFound, YTOutputFormatNotImplemented, \
```

Comment:
`YTOutputFormatNotImplemented` doesn't exist, I think it should be `YTParticleOutputFormatNotImplemented`.

Reply:
You're right, corrected

Reply:
That's fixed now.